gbpservice/nfp/common/constants.py

```diff
@@ -42,6 +42,13 @@
 PENDING_DELETE = "PENDING_DELETE"
 ERROR = "ERROR"
 
+# Constants to extend status strings in neutron.plugins.common.constants
+ONLINE = 'ONLINE'
```

Comment:
@pkharat suggested move these constants to gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py itself.

Will fix that.